test/uk/gov/hmrc/excisemovementcontrolsystemapi/connector/MovementMessageConnectorSpec.scala

```diff
@@ -123,6 +122,14 @@ class MovementMessageConnectorSpec extends PlaySpec with BeforeAndAfterEach with
       result.left.value mustBe BadRequest("any error")
     }
 
+    "return 500 if post request fail" in {
+      when(mockHttpClient.POST[Any, Any](any, any, any)(any, any, any, any))
+        .thenReturn(Future.failed(new RuntimeException("error")))
+      val result = await(submitExciseMovement)
+
+      result.left.value mustBe InternalServerError(s"""{"dateTime":"2023-09-17T09:32:50.345","message":"Exception","debugMessage":"error","emcsCorrelationId":"$emcsCorrelationId"}""")
```

Comment:
Do we have an object we can use to create this? Just might be easier if we change things